src/ansys/mechanical/core/embedding/app.py

```diff
@@ -71,6 +74,19 @@ def __init__(self, db_file=None, **kwargs):
         if self._version == None:
             self._version = _get_default_version()
         initializer.initialize(self._version)
+
+        pkgs = freeze.freeze()
```

Comment:
Add some inline comment saying the reasons for having this code snippet
```suggestion
        # Check if 'pythonnet' is installed... and if so, throw warning
        pkgs = freeze.freeze()
```